guides/process/people/capability_procedure.md

```diff
@@ -0,0 +1,24 @@
+# Capability Procedure
+
+In cases where employees are not meeting the expectations that have been clearly defined to them, and after an expectation health check has been performed, we follow the [Acas Code of Practice on Discipline Procedures](http://www.acas.org.uk/publications).
+
+## Expectation Health Check
+
+Before we escalate to formal proceedings, an expectation health check will be performed by a Director. Capability Procedures usually include measures to identify and address issues, our expectation health check seeks to address issues before reaching that stage.
+
+In the case where a health check has failed to address issues raised, a formal performance review procedure will be followed.
+
+## Formal Performance Review
+
+- Step 1: If it is decided a formal performance review must be carried out, Made Tech will notify the employee in writing of the reasons why and be invited to a meeting to discuss the problem.
+- Step 2: Made Tech will then hold the meeting to discuss the problem, allowing the employee to present their case. The employee has the right to bring a companion* with them.
```

Comment:
Does this * indicate a footnote? I can't see one.

Reply:
You're both right!